src/main/js/EditorPage.jsx

```diff
@@ -378,7 +378,7 @@ class PipelineLoader extends React.Component {
                       "path": "Jenkinsfile",
                       branch: saveToBranch || this.defaultBranch,
                       sourceBranch: branch,
-                      repo: repo,
+                      repo: repo ? repo : team, // if no repo, this is not in an org folder
```

Comment:
the save API needs a repo name, if it's not org folder, it is the name of the `team` (aka org, sorry this is a bit confusing, the terminology has been changing)... this also will eventually go away when we get away from org folders